### PR TITLE
+add css for new svg in the table

### PR DESCRIPTION
**Pull request by pochretien**

### CODE REVIEW

File: scss/tables/table.scss [hunk 1]
```diff
@@ -61,41 +61,41 @@ table {
...
-              opacity: 0.6;
+            .asc-arrow, .desc-arrow {
+              cursor: pointer;
+              opacity: 0.3;
             }
 
-            + i {
-              margin-top: 2px;
+            .asc-arrow, .desc-arrow {
+              &:hover {
```
Comment (marilynechan) [inline scss/tables/table.scss]: Shouldn't this go after line 75? You repeat `.asc-arrow, .desc-arrow` here.

File: scss/tables/table.scss [hunk 2]
```diff
@@ -61,41 +61,41 @@ table {
...
+          display: inline-flex;
 
-          i {
-            position: relative;
-            top: 3px;
-            left: -11px;
-
-            display: block;
+          .coveo-tables-sort {
+            position: absolute;
```
Comment (marilynechan) [inline scss/tables/table.scss]: Is it really necessary to position this with absolute?

File: scss/tables/table.scss [hunk 3]
```diff
@@ -61,41 +61,41 @@ table {
...
-          top: 8px;
+          display: inline-flex;
 
-          i {
-            position: relative;
-            top: 3px;
-            left: -11px;
-
-            display: block;
+          .coveo-tables-sort {
```
Comment (marilynechan) [inline scss/tables/table.scss]: Avoid the coveo or admin prefix, particularly in the styleguide!

File: scss/tables/table.scss (hunk 2 above)
Comment (pochretien) [inline scss/tables/table.scss]: I need to set absolute if a want to make the sort icon with an height higher than the row. That's not all the header which are the same height

File: scss/tables/table.scss [hunk 4]
```diff
@@ -61,41 +61,39 @@ table {
...
-            left: -11px;
-
-            display: block;
+          .tables-sort {
+            position: absolute;
+            top: 25%;
 
-            opacity: 0.3;
+            width: 23px;
+            height: 16px;
```
Comment (marilynechan) [inline scss/tables/table.scss]: Have you tried using standard icon sizing classes (just `icon` or with `mod-lg` modifier)?
There should be no need for absolute positioning if you display the admin-sort-icon inline.

Comment (wfortin) [inline scss/tables/table.scss]: +1

File: scss/tables/fixed-header-table.scss [hunk 5]
```diff
@@ -1,7 +1,7 @@
 .fixed-header-table-container {
   position: relative;
 
-  padding-top: 27px;
+  padding-top: 32px;
```
Comment (marilynechan) [inline scss/tables/fixed-header-table.scss]: You should have a variable in `variables.scss` for this, you use this value at more than 1 place.

File: scss/variables.scss [hunk 6]
```diff
@@ -13,7 +13,7 @@ $filter-box-height: 32px;
 // Header
 $header-height: 65px;
 $header-application-offset-y: 6px;
-$header-padding: 40px;
+$header-padding: 35px;
```
Comment (marilynechan) [inline scss/variables.scss]: AFAIK, this didn't change in the mockups. It should stay 40px.

File: scss/tables/table.scss [hunk 7]
```diff
@@ -146,6 +136,10 @@ table {
 
       border-bottom: $table-border;
 
+      &:first-child {
+        border-left: $table-selected-border-width solid transparent;
```
Comment (marilynechan) [inline scss/tables/table.scss]: Nice, way better :-) .

**Comment by marilynechan** [comment]
Looks good to me, ship it!

**Comment by wfortin** [comment]
:shipit: GO

**Pull request merged**